nautobot/core/testing/integration.py

```diff
@@ -28,23 +28,39 @@ class ObjectsListMixin:
     """
 
     def select_all_items(self):
-        self.browser.find_by_xpath('//*[@id="object_list_form"]//input[@class="toggle"]').click()
+        self.browser.find_by_css("#object_list_form input.toggle").click()
```

Comment:
TIL `find_by_css`. Definitely seems more user-friendly than `find_by_xpath` in these cases!